### PR TITLE
Translate "password_requirements" into Japanese

### PR DESCRIPTION
**Pull request by kachick**

### CODE REVIEW

**Review by mrfinch — COMMENTED** [review]
can you port your changes to ja.coffee?

Sorry for the delay

**Comment by kachick** [comment]
Sure, updated!

How about current diff?

File: app/locale/ja.coffee [hunk 1]
```diff
@@ -651,7 +651,7 @@ module.exports = nativeDescription: "日本語", englishDescription: "Japanese",
 #    already_have_account: "Already have a CodeCombat or Ozaria account"
 #    select_your_role: "Select your role"
 #    select_your_state: "Select your state"
-#    password_requirements: "4 to 64 characters with no repeating"
```
Comment (mrfinch) [inline app/locale/ja.coffee:654]: i think u removed 1 extra space

Comment (kachick) [reply app/locale/ja.coffee:654]: Oops! Sorry, fixed in https://github.com/codecombat/codecombat/pull/6633/commits/b373a9a9bb7ab936cc8c45524edb3ce8681a3866 🙇‍♂️

**Review by mrfinch — APPROVED** [review]

**Comment by kachick** [comment]
Thank you!

**Pull request merged**